lib/Doctrine/Record/Filter.php

```diff
@@ -45,23 +45,32 @@ public function getTable()
         return $this->_table;
     }
 
+    /**
+     * @throws Doctrine_Table_Exception
+     */
```

Comment:
```suggestion
```

An empty code does not throw an exception ;-)
Maybe a sub-class does?

Reply:
It is about the substitution principle.

Does throwing a specific exception is part of the interface? I answered with yes, That's why I put this there.

But, looking deeper now from near usage, it does not. We can say, no exception on the interface means may throw a base one (`Exception` or `Throwable`).

https://github.com/FriendsOfSymfony1/doctrine1/blob/cf27b7a/lib/Doctrine/Table.php#L2706-L2721

Then on `Record::setUp`

https://github.com/FriendsOfSymfony1/doctrine1/blob/e487ff0/lib/Doctrine/Record.php#L297-L305

wdyt?

Reply:
Better to remove it for now
I would add it to an interface it throwing is a contract behavior